lang/ql/projects.txt

```diff
@@ -0,0 +1,5 @@
+# Git URLs of publicly-accessible projects to be used for parsing stats,
+# one per line.
+#
+
+https://github.com/github/codeql
```

Comment:
https://github.com/trailofbits/codeql-queries If you're interested in having more than just one repo